backend/src/main/kotlin/blue/mild/covid/vaxx/service/ValidationService.kt

```diff
@@ -164,8 +168,17 @@ class ValidationService(private val questionService: QuestionService) {
         }
     }
 
-    private fun isPhoneNumberValid(phoneNumber: String): Boolean =
-        """^\+\d{12}$""".toRegex() matches phoneNumber
+    @Suppress("EmptyCatchBlock")
```

Comment:
tady se ti kotlin snazi rict, ze je to osklive 😃 

`runCatching { .. }.getOrNull() ?: false` na celou tuhle fci